Test/PerformanceCollector/FunctionalTests/CommonTests.cs

```diff
@@ -35,6 +35,14 @@ public static void NonExistentCounter(HttpListenerObservable listener)
             AssertCustomCounterReported(counterItems, @"Custom counter - does not exist", false);
         }
 
+        public static void NonExistentCounterWhichUsesPlaceHolder(HttpListenerObservable listener)
+        {
+            var counterItems = listener.ReceiveItemsOfType<TelemetryItem<MetricData>>(10, TestListenerWaitTimeInMs);
```

Comment:
Thats the way all the existing tests are written in that module - if i change one I need to change it everywhere to look consistent.
(will deal with styling separately from this bug fix)